elki/src/test/java/elki/math/statistics/dependence/MCDEDependenceMeasureTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * This file is part of ELKI:
+ * Environment for Developing KDD-Applications Supported by Index-Structures
+ *
+ * Copyright (C) 2019
+ * ELKI Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package elki.math.statistics.dependence;
+
+import elki.utilities.ELKIBuilder;
+import elki.utilities.datastructures.arraylike.NumberArrayAdapter;
+import elki.utilities.datastructures.arraylike.DoubleArrayAdapter;
+import elki.utilities.random.RandomFactory;
+import elki.math.statistics.tests.mcde.*;
+import java.util.Random;
+
+import static elki.math.statistics.dependence.MCDEDependenceMeasure.Par.TEST_ID;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static elki.math.statistics.dependence.MCDEDependenceMeasure.Par.M_ID;
+
+import org.junit.Test;
+
+/**
+ * @author Alan Mazankiewicz
+ * @author Edouard Fouché
+ */
+
+public class MCDEDependenceMeasureTest {
+
+  @Test public void runTest() {
+
+    // independent
+    double[] indi1 = new double[1000];
+    double[] indi2 = new double[1000];
+
+    RandomFactory rnd = new RandomFactory(0);
```

Comment:
In unit tests, because these are not parameterizable, it is fine to use `new Random(0L)` to fix the random seed.
`RandomFactory` is a construct aimed at making it easier to globally fix the random seeds for benchmarking.